lib/signalfx/tracing/span.rb

```diff
@@ -0,0 +1,24 @@
+require 'jaeger/span'
+
+module SignalFx
+    module Tracing
+        module Span
+            @@max_attr_length = (ENV['SIGNALFX_RECORDED_VALUE_MAX_LENGTH'] || '1200').to_i
+
+            def set_tag(key, value)
```

Comment:
Ideally this would include log field map truncation as well, and patching `ThriftTagBuilder.build` may be able to satisfy both without only supporting strings (it calls `.to_s` for non-num/bools).

Reply:
@rmfitzpatrick Updated the patch to support `log_kv` method as well. ThriftTagBuilder could possibly be used else where and not sure if we should patch that directly. Also the method isn't as clean to patch as the span methods it seems.

Reply:
On second thought, it probably gonna be extremely rare that someone uses tag builder directly in combination with `SIGNALFX_RECORDED_VALUE_MAX_LENGTH` env var and it's OK if they do as the behavior will be document.

Reply:
So changing the implementation again to patch tag builder this time.

Reply:
Thanks for the tip.